sql/index.go

```diff
@@ -40,8 +40,8 @@ type Index interface {
 	Database() string
 	// Table returns the table name this index belongs to.
 	Table() string
-	// Expression returns the indexed expression.
-	Expression() Expression
+	// Expressions returns the indexed expressions.
```

Comment:
Can you add here all the explanation about if there is more than one expression they are Columns and so on?